File: ComplementApp-SPA/src/app/_dto/detalleFormatoSolicitudPagoDto.ts
import { ValorSeleccion } from './valorSeleccion';

export interface DetalleFormatoSolicitudPagoDto {
  detalleFormatoSolicitudPagoId: number;
  formatoSolicitudPagoId: number;
  rubroPresupuestal: ValorSeleccion;
  valorAPagar: number;
  dependencia: string;
}

File: ComplementApp.API/Dtos/DetalleFormatoSolicitudPagoDto.cs
namespace ComplementApp.API.Dtos
{
    public class DetalleFormatoSolicitudPagoDto
    {
        public int DetalleFormatoSolicitudPagoId { get; set; }
        public int FormatoSolicitudPagoId { get; set; }
        public ValorSeleccion RubroPresupuestal { get; set; }
        public decimal ValorAPagar { get; set; }
        public string Dependencia { get; set; }
        public int ClavePresupuestalContableId { get; set; }
    }
}